Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.12
+
+WORKDIR /app
+
+# Install dependencies
+COPY requirements.txt /requirements.txt
+RUN pip install --upgrade pip
+RUN pip install -r /requirements.txt
+
+# Copy project
+COPY . /app
+
+# Set environment variables
+ENV FLASK_APP=flaskr
+ENV FLASK_ENV=development
```

Comment:
`FLASK_ENV` shouldn't be set here but should instead be added as a parameter in the architect.yml file so that it's configurable. The app should also be able to be run in a production mode for remote deployments